backend/scripts/pull-from-idol-images.ts

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-console */
+import { writeFile } from 'fs/promises';
+import { join } from 'path';
+import { allMemberImages } from '../src/API/imageAPI';
+
+const main = async () => {
+  console.log('Downloading all member images...');
+  const processedImages = await allMemberImages();
```

Comment:
what about profile images for non-active members? aren't they also gonna get pulled in?

Reply:
Yep!

Reply:
all of them? riya's image is probably in there still and i'm guessing it isn't planned for hers to be on there

Reply:
I did a clean-up recently so they're just the same images. I'll add a follow up item to autocleanup that bucket if we no longer need an image, or at the very least doing filtering here.

Reply:
https://www.notion.so/Make-Pull-from-IDOL-Images-Script-Only-Download-Necessary-Images-for-Website-1470ad723ce1807ea3fcd11cabb49138?pvs=4